autogluon/task/tabular_prediction/hyperparameter_configs.py

```diff
@@ -0,0 +1,49 @@
+
+import copy
+
+
+# Dictionary of preset hyperparameter configurations.
+hyperparameter_config_dict = dict(
+    # Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
+    default={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+        'RF': {},
+        'XT': {},
+        'KNN': {},
+        'custom': ['GBM'],
+    },
+    # Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
+    light={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+        'RF': {'max_depth': 15},
+        'XT': {'max_depth': 15},
+        'custom': ['GBM'],
+    },
+    # Results in much smaller models. Behaves similarly to 'light', but in many cases with over 10x less disk usage and a further reduction in accuracy.
+    very_light={
```

Comment:
Do we want to constrain max_size of weightedensemble in very_light?
I guess it's already constrained without HPO since there's only 3 models...

Reply:
Ditto of previous comment on light.